src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpExtractInterfaceDialog.cs

```diff
@@ -142,5 +142,146 @@ public void OnlySelectedItemsAreGenerated()
     void M2();
 }");
         }
+
+        [WpfFact, Trait(Traits.Feature, Traits.Features.CodeActionsExtractInterface)]
+        public void CheckSameFile()
+        {
+            SetUpEditor(@"class C$$
```

Comment:
Unit tests exist for the ViewModel, but only test the constructor functionality. Since this is using XAML binding, I don't think it's easily testable on that front with UTs. Everything else seems to be done with integration or functional tests, which is what I added to. I believe this is the right call